src/pieces.rs

```diff
@@ -89,24 +54,18 @@ impl<'a> Tower<'a> {
     ///    For example, (Your) Pawn, (Your) Gold, (Your) Gold is disallowed
     ///    but (Your) Pawn, (Your) Gold, (Enemy) Gold is fine
```

Comment:
I can't seem to figure out how to rust doc tests on binary projects.

Looking online, it seems that rust does not support doctests in binary projects
See https://doc.rust-lang.org/1.6.0/book/documentation.html
> However, cargo test will not test binary crates, only library ones. This is due to the way rustdoc works: it links against the library to be tested, but with a binary, there’s nothing to link to.: